projects/ccf-rui/src/app/shared/components/opacity-slider/opacity-slider.component.scss

```diff
@@ -0,0 +1,55 @@
+:host {
+  height: 2.25rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 !important;
```

Comment:
Something is wrong elsewhere otherwise you wouldn't need `!important`. Find and fix!